project-api/project-api-test/src/test/java/io/ballerina/projects/test/plugins/IDLClientCompilerTests.java

```diff
@@ -62,7 +62,7 @@ public class IDLClientCompilerTests {
     private static final String UNSUPPORTED_EXPOSURE_OF_PUBLIC_CONSTRUCT_ERROR =
             "exposing a construct from a module generated for a client declaration is not yet supported";
     private static final String NO_CLIENT_OBJECT_NAMED_CLIENT_IN_GENERATED_MODULE_ERROR =
-            "a module generated for a client declaration must have an object type or class named 'client'";
+            "a module generated for a client declaration must have an object type or class named 'Client'";
```

Comment:
We need to change the generated client used for this test class: https://github.com/ballerina-platform/ballerina-lang/blob/d956272741ca0b2f5b765a729e8fcf0f0304f047/project-api/test-artifacts/idl-import-simple-compiler-plugin/src/main/java/io/idl/plugins/simpleclient/SimpleClientGeneratorPlugin.java#L101

Reply:
will change

Reply:
Updated